README.md

```diff
@@ -102,6 +103,7 @@ These are the platforms we must support (must pass end-to-end testing):
 - Trusty
 - CentOS6
 - CentOS7
+- CentOS8
```

Comment:
Take a look at Ansible meta files https://github.com/StackStorm/ansible-st2/blob/master/roles/StackStorm.st2/meta/main.yml#L13-L16 where new `EL8` could be added too.